bin/bossSwitch.py

```diff
@@ -0,0 +1,210 @@
+    """
+    with open('../'+ name + '.pkl', 'wb') as f:
+        pickle.dump(obj, f, pickle.HIGHEST_PROTOCOL)
+
+def load_obj(name):
+    """
+        Method to load saved objects in .pkl file
+
+        Args:
+            name : The .pkl file name to open
+        
+        Returns:
+            object saved within .pkl file
+    """
+    with open('../' + name + '.pkl', 'rb') as f:
+        return pickle.load(f)
+
+class bcolors:
+    """
+        Used to add coloring to terminal print statements
+    """
+    OKBLUE = '\033[94m'
+    OKGREEN = '\033[92m'
+    WARNING = '\033[93m'
+    FAIL = '\033[91m'
+    ENDC = '\033[0m'
+
+if __name__ == '__main__':
+
+    def create_help(header, options):
+        """Create formated help."""
+        return "\n" + header + "\n" + \
+               "\n".join(map(lambda x: "  " + x, options)) + "\n"
+
+    actions = ["on", "off"]
+    actions_help = create_help("action supports the following:", actions)
+
+    scenarios = ["development", "production"]
+    scenario_help = create_help("scenario supports the following:", scenarios)
+
+    parser = argparse.ArgumentParser(description = "Script to turn the boss on and off by stopping and restarting EC2 instances.",
+                                     formatter_class=argparse.RawDescriptionHelpFormatter,
+                                     epilog=actions_help + scenario_help)
+    parser.add_argument("--aws-credentials", "-a",
+                        metavar = "<file>",
+                        default = "../config/aws-credentials",
+                        help = "File with credentials to use when connecting to AWS (default: AWS_CREDENTIALS)")
+    parser.add_argument("--scenario",
+                        metavar = "<scenario>",
+                        default = "development",
+                        choices = scenarios,
+                        help = "The deployment configuration to use when creating the stack (instance size, autoscale group size, etc) (default: development)")
+    parser.add_argument("action",
+                        choices = actions,
+                        metavar = "action",
+                        help = "Action to execute")
+    parser.add_argument("vpc",
+                    metavar = "vpc-machine-name",
+                    help = "The vault machine name. ex: vault.user.boss")
+    args = parser.parse_args()
+
+    #Loading AWS configuration files.
+    creds = json.load(open(args.aws_credentials))
+    aws_access_key_id = creds["aws_access_key"]
+    aws_secret_access_key = creds["aws_secret_key"]
+    region_name = 'us-east-1'
+
+    if creds is None:
+        print("Error: AWS credentials not provided and AWS_CREDENTIALS is not defined")
+
+    # specify AWS keys, sets up connection to the client.
+    auth = {"aws_access_key_id": aws_access_key_id, "aws_secret_access_key": aws_secret_access_key, "region_name": region_name}
+    client = boto3.client('autoscaling', **auth)
```

Comment:
It would be cleaner to write `client = aws.create_sessions(creds).client('autoscaling')`